iocore/net/SSLUtils.cc

```diff
@@ -1866,10 +1934,17 @@ ssl_store_ssl_context(const SSLConfigParams *params, SSLCertLookup *lookup, cons
     keyblock = ssl_context_enable_tickets(ctx, NULL);
   }
 
+  // Generate HPKP header if hpkp is enabled.
+  if (sslMultCertSettings.hpkp_enabled >= 0 ? sslMultCertSettings.hpkp_enabled : params->hpkp_enabled) {
```

Comment:
So it is not possible to enable HPKP globally then disable it on specific certificates? I think the expected behavior is for the certificate settings to always have precedence (if they are specified).  See [TS-2773](https://issues.apache.org/jira/browse/TS-2773) for